app/models/vm.rb

```diff
@@ -4,6 +4,10 @@ class Vm < VmOrTemplate
   has_one :container_deployment_node
   include_concern 'Operations'
 
+  extend InterRegionApiMethodRelay
+  api_relay_method :start
+  api_relay_method :stop
```

Comment:
I actually tried putting them with the definitions, but ran into some issues with extending the new module in a concern (I don't think [this](https://github.com/ManageIQ/manageiq/pull/11782/files#diff-8c984f040f95783edcff073724877877R73) would work properly in the `Operations` concern where the `start` and `stop` methods are defined.)